sourcecode/apis/contentauthor/resources/assets/react/components/QuestionContentType/components/QuestionCard/components/Image/ImageLayout.js

```diff
@@ -21,14 +21,15 @@ function ImageLayout(props) {
 
     let icon = null;
     if ( previewImage === null) {
-        icon = <ImageIcon />;
+        icon = <ImageIcon style={{cursor: 'pointer'}} />;
         if ( readOnly === false ) {
             icon = (
                 <DropZone
                     onDropAccepted={onDrop}
                     multiple={false}
                     className="imageDropzone"
                     disabled={readOnly}
+                    aria-label="Add image"
```

Comment:
The button is the `<input type="file">` created by the DropZone component. The `aria-label` must be placed on that.

<img width="532" alt="image" src="https://user-images.githubusercontent.com/84008144/236475135-858b7787-f227-459b-91bb-a9cc95ce7140.png">

When using keyboard navigation, the button also has no indication it is focused.